### PR TITLE
downloadable link

### PR DESCRIPTION
**Pull request by flutistar**
#23264

- [x] Downloadable link on Document Indexing Review Form
- [x] Sort after load new page

image

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://documents-ui-dev--pr-73-vxyfsma1.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://documents-ui-dev--pr-73-vxyfsma1.web.app

**Review by doug-lovett — APPROVED** [review]
Looking good.

**Review by cameron-eyds — APPROVED** [review]
Fantastic, well done. Super minor comment

File: document-service/documents-ui/src/pages/DocumentIndexingReview.vue [hunk 1]
```diff
@@ -39,63 +46,97 @@ const {
...
+                          >{{ document.name }}</a
+                        >
                       </ol>
                     </div>
                   </div>
-              </div>
-            </div></template>
+                </div>
+              </div></template
+            >
```
Comment (cameron-eyds) [inline document-service/documents-ui/src/pages/DocumentIndexingReview.vue:136]: Minor new line fix here

**Pull request merged**